### PR TITLE
Javascript/javascript1/week2

### PR DESCRIPTION
**Pull request by dinythomas89**
Added html and 4 js files

### CODE REVIEW

File: javascript/javascript1/week2/student-manager.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+addStudentToClass('willam');
+addStudentToClass('Jack');
+addStudentToClass('kate');
+addStudentToClass('Tomas');
+addStudentToClass('Queen');
+addStudentToClass('Bella');
+console.log(class07Students);
+
+
+function getNumberOfStudents() {
```
Comment (flom84) [inline javascript/javascript1/week2/student-manager.js]: I believe that your intention is to print the number of students. Also you don't  use this function anywhere in your code.
```suggestion
function printNumberOfStudents() {
```

File: javascript/javascript1/week2/student-manager.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+const class07Students = [];
+function addStudentToClass(studentName) { 
+    if(studentName==='Queen') {
+        class07Students.push(studentName);
+    }
+  else if(studentName===''){
+      console.log('Please enter a valid name');
+  }
+  
+  else if(class07Students.length>=6){
```
Comment (flom84) [inline javascript/javascript1/week2/student-manager.js]:
```suggestion
  else if(class07Students.length>6){
```

File: javascript/javascript1/week2/student-manager.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+      console.log('Cannot add more students to class 07');
+  }
+  else if(toCheckStudentExist(studentName)){
+      console.log('Student '+ studentName+' is already in the class');
+  }  
+  else{
+      class07Students.push(studentName)
+      return class07Students;
```
Comment (flom84) [inline javascript/javascript1/week2/student-manager.js]: you already have declared your array global...therefore you don't need to return.
```suggestion
```

File: javascript/javascript1/week2/flight-booking.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+    return firstName+' '+surName;
+}
+let fullName1=getFullName("Benjamin", "Hughes");
+let fullName2=getFullName("Iza", "David");
+console.log(fullName1);
+console.log(fullName2);
+
+//formal fullname
+function getFormalFullname(firstName, surName,useFormalName=true,gender){
+    if((firstName && surName)==='')
```
Comment (flom84) [inline javascript/javascript1/week2/flight-booking.js]: `(firstName && surName)===''
`
JavaScript would evaluate this expression as follow

1. (firstName && surName) -> true or false (boolean value)
2.  true[false] === ' ' - > this is always false.
I believe your intention is not to have this expression to evaluate `false` all the time

File: javascript/javascript1/week2/student-manager.js (hunk 1 above)
Comment (dinythomas89) [reply javascript/javascript1/week2/student-manager.js]: Changed the function to printNumberOfStudents(). In homework this function ask us to just display the number of students

File: javascript/javascript1/week2/student-manager.js (hunk 2 above)
Comment (dinythomas89) [reply javascript/javascript1/week2/student-manager.js]: I changed it to - else if(class07Students.length>5) -  as i can only have max of length 6.

File: javascript/javascript1/week2/student-manager.js (hunk 3 above)
Comment (dinythomas89) [reply javascript/javascript1/week2/student-manager.js]: I removed the return statement.

File: javascript/javascript1/week2/flight-booking.js (hunk 4 above)
Comment (dinythomas89) [reply javascript/javascript1/week2/flight-booking.js]: I changed it to (firstName==='' || surName==='')

File: javascript/javascript1/week2/student-manager.js (hunk 2 above)
Comment (flom84) [reply javascript/javascript1/week2/student-manager.js]: sorry. i ment 6

**Pull request merged**